culane_metric/evaluate.py

```diff
@@ -85,8 +85,24 @@ def continuous_cross_iou(xs, ys, width=30):
     return ious
 
 
+def remove_con_dup(x):
+    """Customize a set op to remove consecutive duplications"""
```

Comment:
"""Removes conescutive duplicates"""
is more concise, same meaning. "Customize a set op" does not seem to quite fit.